doc/GUIDE.md

```diff
@@ -737,97 +542,37 @@ Haskell packages, this will often be enough to build most packages. However,
 at times, you may find that not all dependencies required may be available in
 the Stackage snapshots.
 
-Let's simulate an unsatisfied dependency by adding acme-missiles to our
-build-depends and re-initing:
+Let's simulate an unsatisfied dependency by adding acme-missiles to the `.cabal` file
+(yackage does not currently support hpack, but you can also [hpack-convert](https://github.com/yamadapc/hpack-convert)
```

Comment:
:( that's unfortunate. I wonder if we can fix that. Think a PR would be appropriate? The reason for changing to hpack seems a bit flimsy though, for an actual piece of software to have to change just to fit as a decent example in some other documentation ...

Or maybe it's a decent learning opportunity anyway, since they'll probably have to deal with at least some packages eventually that don't have a package.yaml?

Probably beyond the scope of this documentation change anyway though.